src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int maxValueOfRandomDigit = 100;

```

Comment:
Violates constant naming convention
```suggestion
    private static final int maxValueOfRandomDigit = 100;

```

Reply:
What breaks the constant naming convention? exactly what's wrong